src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -243,3 +258,31 @@ export const showDialogUrgCare = currentQuery => {
 
   return null;
 };
+
+/**
+ * Helper fn to record Markers PIN action events for GA
+ */
+export const recordMarkerEvents = r => {
+  const { classification, name, facilityType } = r.attributes;
+  const distance = r.distance;
+  recordEvent({ event: 'fl-map-pin-click' });
+  if (classification && name && facilityType && distance) {
+    recordEvent({ 'fl-facility-type': facilityType });
+    recordEvent({ 'fl-facility-classification': classification });
+    recordEvent({ 'fl-facility-name': name });
+    recordEvent({ 'fl-facility-distance-from-search': distance });
+  }
+};
+
+/**
+ * Helper fn to record map movement after search
+ */
+export const recordDistanceSearchMove = (searchCoords, center) => {
+  const distanceAfterMove = distBetween(
+    searchCoords.lat,
+    searchCoords.lng,
+    center.lat,
+    center.lng,
+  );
```

Comment:
If `distBetween` is very small, or zero, should we still call `recordEvent`?

Reply:
this is question for analytics guys or Michelle,  because distance can be in decimals

Reply:
but I still have to work on this event as it does not seem to be updating as I thought

Reply:
It looks like this will be triggered every time the bounds are changed, even on zoom with no pan. So if `distBetween` is 0 I would suggest not triggering `'fl-map-miles-moved``.